tests/test_exist_client.py

```diff
@@ -75,3 +77,14 @@ def test_url_parsing(url, properties):
     assert client.host == properties[3]
     assert client.port == properties[4]
     assert client.prefix == properties[5]
+
+
+@pytest.mark.parametrize("text_length", [7000, 8000, 9000, 10000])
```

Comment:
what about just testing *one* really large document from the real world and compare that bytewise for confirmation? the `dada_manifest.xml` is just a little larger than 8k which is a widespread default as GET length limit in webservers. but it could be cloned some times and wrapped under one node to form a test document.

Reply:
That's an interesting fact, I wasn't aware of this specific limit until now. I agree regarding the test, I'll do as you suggested.

Reply:
It seems to me that there's an issue with this approach though: We need a big unfragmented chunk of text in order to include it in a simple query expression. I don't know how we would use the manifesto to build such a long expression ina meaningful way. Sure, I could take one paragraph and create a nonsensical XPath expression by repeating it (e. g. `//p[contains(., "...") | contains(., "...") | contains(., "...")]`) and make it long enough to satisfy the requirements of the test, but this feels like it defeats the purpose of using a real world document, so why not just use the boring monotonous string after all... I hit a dilemma is what I'm trying to say :woozy_face:

Reply:
I'll just try creating that big chunk of text by putting the `.full_text` value of the entire manifesto root node several times in a test document and then use that.

Reply:
sorry, my bad, i was assuming that the test is about storing a large document. :-/